setup.py

```diff
@@ -45,7 +45,7 @@ def required(requirements_file):
         requirements = f.read().splitlines()
         if 'MYCROFT_LOOSE_REQUIREMENTS' in os.environ:
             print('USING LOOSE REQUIREMENTS!')
-            requirements = [r.replace('==', '>=') for r in requirements]
+            requirements = [r.replace('==', '>=').replace('~=', '>=') for r in requirements]
```

Comment:
Are "Loose requirements" something we want to support at all? Seems like properly spec'd dependencies are the "right way" to handle this..

Reply:
this was requested by @PureTryOut he needs it for packaging in Alpine, i assume it will also be useful for other distros and even for some users. If people know what they are doing i think this is perfectly fine and useful, it's not like it can be used by accident

Reply:
I kind of agree with Daniel, but the support itself is not introduced in this PR. This just narrows the implications a bit. 🤷‍♂️